build.sbt

```diff
@@ -190,9 +190,12 @@ lazy val coreJava = (project in file("core-java"))
       val inputDir = (rdfProtosJava / target).value / ("scala-" + scalaVersion.value) /
         "src_managed" / "main" / "compiled_protobuf" /
         "eu" / "neverblink" / "jelly" / "core" / "proto" / "v1"
+
       val outputDir = sourceManaged.value / "main" /
         "eu" / "neverblink" / "jelly" / "core" / "proto" / "v1"
-      val javaFiles = (inputDir * "*.java").get
+
+      val javaFiles = (inputDir * "*.java").get ++ (inputDir * "**" * "*.java").get
```

Comment:
Nooo, don't do that. The `patch` proto classes should be included in `core`. Only in `core-patch`. This is important, we don't want to bloat the core module.

Reply:
*should not be included in `core`